dev/source/docs/roadmap.rst

```diff
@@ -1,120 +1,118 @@
 
-Scripting (MichaelDB)
----------------------
+- ESC feedback handling (Randy)
+- 4kHz+ loop rate PIDs (Tridge, Leonard)
 
-- add Lua scripting to ArduPilot (`video <https://www.youtube.com/watch?v=ZUNOZMxOwsI>`_ from 2019 un-conference)
+- Attitude controller
 
-EKF (PaulR / Randy)
--------------------
+  - Enable rate loop saturation from external sources (Leonard)
+  - SI unit input (Leonard)
 
-- Robust yaw estimator for Copters
-- SLAM integration for position estimation
+- Position Controller
 
+  - Update Z to XY feed-forward architecture (Leonard)
+  - Include position error limits based on velocity saturation (Leonard)
+  - Include velocity error limits based on accel saturation (Leonard)
+  - Handle EKF reset for the position correctly (Leonard)
+  - Baseline Velocity input (Leonard)
+  
+- S-Curve
+  - Concept demonstrator done. Implementation needed. (Leonard)
+		
+- Guided Mode
+  - Input shaped using FF (Leonard)
+		
+- AutoTune
+  - Add tuning type to PID object (Bill Geyer)
+  
+  
 Plane (Tridge / MichaelDB)
 --------------------------
 
-- Takeoff mode
-- Automatic Taxiing
+- Automatic Taxing (Tridge)
+- Better RealFlight quadplane model
 - Increase QuadPlane landing approach options
-- Plane architectural improvements (aka "the onion")
+- Plane architectural improvements (aka “the onion” v2)
 - QuadPlane flat/spoiler control (MichaelDB)
+  
 
-Non-GPS Navigation, Avoidance and Path Planning (Randy / Patrick Poirier)
--------------------------------------------------------------------------
-
-- Add support for Intel RealSense T265, D435 (ROS & non-ROS)
-- Copter & Rover back away from objects
-- Proximity data (i.e. obstacles) recorded in Earth coordinates
-- On-board Path Planning around obstacles
-- Improve Off-Board SLAM + Object Avoidance (`dev wiki link <https://ardupilot.org/dev/docs/ros-cartographer-slam.html>`__)
-
-Copter (Randy / Leonard / Tridge)
----------------------------------
-
-- ESC feedback handling
-- Flight Mode Improvements
-
-  - System identification mode
-  - 4kHz+ loop rate PIDs
-  - Stand-By mode
-
-- Attitude controller
-
-  - Separate FF and PID input for correct scaling
-  - Rate loop updates
-  - Enable rate loop saturation from external sources
-  - SI unit input
-
-- Position Controller
-
-  - Update Z to XY feed forward architecture
-  - Include position error limits based on velocity saturation
-  - Include velocity error limits based on accel saturation
-  - Handle EKF reset for position correctly
+Rover & Boat
+------------
 
-- AutoTune
+- S-Curve navigation (`video from 2019 un-conference <https://www.youtube.com/watch?v=LHq5o9zgNWk>`__) (Leonard/Randy)
+- Stick Mixing in Auto (TomP)
+- Underwater mapping with scanning sonar (Randy/PeterB)
+- Precision Docking (using marker onshore) (Randy)
+- high speed sailing inprovements (Randy)
+- Roll and pitch control for boats (Randy)
+- AIS (i.e., ADSB for boats) (Randy)
 
-  - Add tuning type to PID object
 
 Trad Heli (Bill Geyer, Chris Olson)
```

Comment:
Let's change the title to be just, "Trad Heli (Bill Geyer)".

Reply:
ok, done.